include/searchresultsformaction.h

```diff
@@ -0,0 +1,43 @@
+#ifndef NEWSBOAT_SEARCHRESULTSFORMACTION_H_
+#define NEWSBOAT_SEARCHRESULTSFORMACTION_H_
+#include "configcontainer.h"
+#include "itemlistformaction.h"
+#include "logger.h"
```

Comment:
Do we need logger.h in here? It doesn't look like this header logs anything.

Reply:
https://github.com/newsboat/newsboat/pull/2031/commits/30cc562a54975d740a046c25d556d1e930d46e49